### PR TITLE
Move extension queries into controller

### PR DESCRIPTION
**Pull request by sballance**
### Please check if the PR fulfills these requirements:

* [ ] The PR title and message follows our [guidelines](https://submitty.org/developer/how_to_contribute)
* [ ] Tests for the changes have been added/updated (if possible)
* [ ] Documentation has been updated/added if relevant

### CODE REVIEW

File: site/app/controllers/admin/LateController.php [hunk 1]
```diff
@@ -35,10 +35,15 @@ public function viewLateDays() {
...
+        $current_gid = isset($_COOKIE['exception_gid']) ? $_COOKIE['exception_gid'] : null;
+        $users = $this->core->getQueries()->getUsersWithExtensions($current_gid);
         return Response::WebOnlyResponse(
             new WebResponse(
                 ['admin', 'Extensions'],
                 'displayExtensions',
+                $students,
+                $users,
                 $this->core->getQueries()->getAllElectronicGradeablesIds()
             )
```
Comment (prestoncarman) [inline site/app/controllers/admin/LateController.php]: The parameter order does not match the view.

**Comment by JarodO** [comment]
Other than what Professor Carman said, this looks good to me.

File: site/app/controllers/admin/LateController.php (hunk 1 above)
Comment (sballance) [reply site/app/controllers/admin/LateController.php]: This has been fixed in my latest commit

**Comment by danielarlt** [comment]
This looks good to me!

**Comment by 3dCameron** [comment]
Looks clean and correct.

**Comment by sballance** [comment]
@3dCameron Is there a reason this was closed without being merged?

**Pull request closed without merging**